model/project.go

```diff
@@ -1519,10 +1519,10 @@ func (p *Project) GetDisplayTask(variant, name string) *patch.DisplayTask {
 	return nil
 }
 
-// FetchVersionsAndAssociatedBuilds is a helper function to fetch a group of versions and their associated builds.
+// FetchVersionsBuildsAndTasks is a helper function to fetch a group of versions and their associated builds and tasks.
 // Returns the versions themselves, as well as a map of version id -> the
 // builds that are a part of the version (unsorted).
-func FetchVersionsAndAssociatedBuilds(project *Project, skip int, numVersions int, showTriggered bool) ([]Version, map[string][]build.Build, error) {
+func FetchVersionsBuildsAndTasks(project *Project, skip int, numVersions int, showTriggered bool) ([]Version, map[string][]build.Build, map[string][]task.Task, error) {
```

Comment:
The new struct isn't super intuitive (for example, the task map could be version -> task or build -> task without reading code), should probably explain this structure by adding to the comment at 1524

Reply:
done